src/sage/rings/qqbar.py

```diff
@@ -2827,6 +2828,69 @@ def number_field_elements_from_algebraics(numbers, minimal=False,
            To:   Algebraic Real Field
            Defn: 1 |--> 1)
 
+    Test ``embedded`` for quadratic and cyclotomic fields::
+
+        sage: number_field_elements_from_algebraics([QQbar((-1)^(2/3))], embedded=False, minimal=True)
+        (Number Field in zeta6 with defining polynomial x^2 - x + 1,
+         [zeta6 - 1],
+         Ring morphism:
+           From: Number Field in zeta6 with defining polynomial x^2 - x + 1
+           To:   Algebraic Field
+           Defn: zeta6 |--> 0.500000000000000? + 0.866025403784439?*I)
+        sage: _[0].coerce_embedding()
```

Comment:
I'm surprised that the underscore would work here.  Even if it does, I think it's clearer to give the return value explicit names.